### PR TITLE
[docs] Test API References

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
\cc @VasilyStrelyaev

### CODE REVIEW

File: docs/articles/documentation/test-api/A-Z.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+This topic contains an alphabetical list of all the test API functions and objects with members.
+
+## Functions
+
+* [afterEach](test-code-structure.html#initialization-and-clean-up)
+* [beforeEach](test-code-structure.html#initialization-and-clean-up)
+* [fixture](test-code-structure.html#fixtures)
+* [page](test-code-structure.html#specifying-the-start-webpage)
+* [test](test-code-structure.html#tests)
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/A-Z.md]: We only have two global functions - `fixture` and `test`.
The `page`, `afterEach` and `beforeEach` functions are exposed by `Fixture` objects in the global scope. (check this with @inikulin). I'm not sure but it seems we have to formally mention this `Fixture` objects in this topic.

File: docs/articles/documentation/test-api/A-Z.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: A-Z
+permalink: /documentation/test-api/a-z.html
+---
+# A-Z
+
+This topic contains an alphabetical list of all the test API functions and objects with members.
+
+## Functions
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/A-Z.md]: I wonder what's right for this - `global functions` or maybe `factory functions` (like `createTestCafe`)?

File: docs/articles/documentation/test-api/A-Z.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+* [afterEach](test-code-structure.html#initialization-and-clean-up)
+* [beforeEach](test-code-structure.html#initialization-and-clean-up)
+* [fixture](test-code-structure.html#fixtures)
+* [page](test-code-structure.html#specifying-the-start-webpage)
+* [test](test-code-structure.html#tests)
+
+## Objects with Members
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/A-Z.md]: Maybe just `Objects`?

File: docs/articles/documentation/test-api/A-Z.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+* [ClientFunction](obtaining-data-from-the-client.html#creating-client-functions)
+    * [with](obtaining-data-from-the-client.html#overwriting-options)
+* [DOM Node Snapshot (with members)](selecting-page-elements/dom-node-snapshots.html)
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/A-Z.md]: Again, `with members` is not quite clear.
Maybe *members* in italics as a single child item?

```
* [DOM Node Snapshot](selecting-page-elements/dom-node-snapshots.html)
    * *[members](selecting-page-elements/dom-node-snapshots.html)*
```

**Comment by MargaritaLoseva** [comment]
\cc @inikulin

File: docs/articles/documentation/test-api/A-Z.md (hunk 1 above)
Comment (MargaritaLoseva) [inline docs/articles/documentation/test-api/A-Z.md]: @inikulin Can we introduce the `Fixture` object to the documentation? Currently, when we describe `page`, `afterEach` and `beforeEach`, we use the _fixture declaration_ term.

**Comment by VasilyStrelyaev** [comment]
4 hours left until it's [semi]automatically merged

File: docs/articles/documentation/test-api/A-Z.md (hunk 1 above)
Comment (inikulin) [inline docs/articles/documentation/test-api/A-Z.md]: > Can we introduce the Fixture object to the documentation?

Yeah, that makes sense

File: docs/README.md [hunk 5]
```diff
@@ -39,6 +39,7 @@
   * [Pausing the Test](articles/documentation/test-api/pausing-the-test.md)
   * [Handling Native Dialogs](articles/documentation/test-api/handling-native-dialogs.md)
   * [Working with &lt;iframes&gt;](articles/documentation/test-api/working-with-iframes.md)
+  * [A-Z](articles/documentation/test-api/a-z.md)
```
Comment (inikulin) [inline docs/README.md]: It looks weird here with this name, maybe we can somehow describe it better in title?

File: docs/articles/documentation/test-api/A-Z.md (hunk 2 above)
Comment (inikulin) [inline docs/articles/documentation/test-api/A-Z.md]: e.g. `afterEach` is not global

File: docs/articles/documentation/test-api/A-Z.md (hunk 3 above)
Comment (inikulin) [inline docs/articles/documentation/test-api/A-Z.md]: Both `Objects` and `Objects with Members` sounds weird. Maybe we'll get rid of this subtitles?

File: docs/README.md (hunk 5 above)
Comment (VasilyStrelyaev) [inline docs/README.md]: We can surely name it `Index` but nothing will be more simple and eye-catching for an alphabetical list than `A-Z`.
This topic is aimed to provide fast reference, for example, when someone gets puzzled with someone else's test code. `A-Z` will stand out from other topics.

Comment (VasilyStrelyaev) [inline docs/README.md]: hmmm... if only `A-Z Index`

**Comment by MargaritaLoseva** [comment]
\cc @DevExpress/testcafe

**Pull request merged**